draft-bbs-signatures.md

```diff
@@ -443,9 +457,9 @@ Procedure:
 
 25. s^ = s~ - c * s'
 
-26. for i in RIdxs: m^[i] = m~[i] - c * msg[i]
+26. for j in (j1, j2,..., jU): m^[j] = m~[j] - c * msg_j
```

Comment:
`m^_j` and `m~_j`? Seems inconsistent to sometimes use the `[n]` notation for array indices.

Reply:
Updated. I agree but if we move to `m_c` instead of `m^` for example then it will become `m_c_j` and `m_c_j1, m_c_j2, ...` which I don't know if I like. updated for now for consistency.

Reply:
I think we can probably keep m~ and m^ now they aren't causing formatting issues.